clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -18,28 +18,49 @@
 import defaults as d
 
 
+def api_call_latency(title, verb, scope, threshold):
+    return d.Graph(
+        title=title,
+        targets=[
+            g.Target(expr=str(threshold), legendFormat="threshold"),
+            g.Target(
+                expr='apiserver:apiserver_request_latency_1m:histogram_quantile{quantile="0.99", verb=~"%(verb)s", scope=~"%(scope)s"}'
+                % {"verb": verb, "scope": scope},
+                legendFormat="{{verb}} {{scope}}/{{resource}}",
+            ),
+        ],
+        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
+    )
+
+
 CLUSTERLOADER_PANELS = [
     d.simple_graph(
         "Requests",
-        'sum(rate(apiserver_request_count{client="clusterloader/v0.0.0 (linux/amd64) kubernetes/$Format", verb!="CONNECT"}[1m])) by (verb, resource)',
-    ),
-    d.simple_graph(
-        "API call latency (1s thresholds)",
-        'apiserver:apiserver_request_latency:histogram_quantile{quantile="0.99", verb!~"CONNECT|LIST|WATCH"}',
-        legend="{{verb}} {{scope}}/{{resource}}",
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
-    ),
-    d.simple_graph(
-        "API call latency aggregated (1s thresholds)",
-        'histogram_quantile(0.99, sum(rate(apiserver_request_duration_seconds_bucket{verb!~"LIST|WATCH|CONNECT|DELETECOLLECTION"}[1d])) by (le, resource, verb, scope, subresource))',
-        legend="{{verb}} {{scope}}/{{resource}}",
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
-    ),
-    d.simple_graph(
-        "API call latency aggregated (prometheus, 1s threshold)",
-        'quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile{verb!~"LIST|WATCH|CONNECT|DELETECOLLECTION"}[1d])',
-        legend="{{verb}} {{scope}}/{{resource}}",
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
+        'sum(rate(apiserver_request_total{client="clusterloader/v0.0.0 (linux/amd64) kubernetes/$Format", verb!="CONNECT"}[1m])) by (verb, resource)',
```

Comment:
sum(irate(apiserver_request_total))?

I was debugging for an hour why my test is overloaded at the very beginning and changing to irate allowed me to discover that I'm creating ~1K QPS of some objects creation.

using rate() was hiding this issue